admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -54,15 +54,34 @@
       {/if}
 
       {if isset($maintenance_mode) && $maintenance_mode == true}
+        {capture name="title"}
+          <p class="text-left text-nowrap">
+            <strong>{l s='Your shop is in maintenance.' d='Admin.Navigation.Notification'}</strong>
+          </p>
+          <p class="text-left">
+              {l s='Your visitors and customers cannot access your shop while in maintenance mode.' d='Admin.Navigation.Notification'}
+          </p>
+          <p class="text-left">
+              {l s='To manage the maintenance settings, go to Shop Parameters > Maintenance tab.' d='Admin.Navigation.Notification'}
+          </p>
+          {if isset($maintenance_allow_admins) && $maintenance_allow_admins}
+            <p class="text-left">
+              {l s='Admins can access the shop front-end without storing their IP.' d='Admin.Navigation.Notification'}
```

Comment:
```suggestion
              {l s='Admins can access the store front-end without storing their IP.' d='Admin.Navigation.Notification'}
```

Reply:
@l-delin I think it should say `front office` not `front-end` right? :)